plugwise_usb/nodes/circle.py

```diff
@@ -297,11 +298,15 @@ async def energy_update(
             self._last_energy_log_requested = await self.energy_log_update(self._last_log_address)
 
         if self._energy_counters.log_rollover:
-            _LOGGER.debug(
+            _LOGGER.info(
                 "async_energy_update | Log rollover for %s",
                 self._node_info.mac,
             )
             if await self.node_info_update():
+                # if the log_address changes big chance we did not see slot 4, request it once
+                if self._previous_log_address:
+                    await self.energy_log_update(self._previous_log_address)
+                    self._previous_log_address = None
```

Comment:
I discovered yesterday the log address at the `NodeInfoRespone` message is actually the pointer to the current log period which is not populated with the counters yet.
See also [this commit in the async branch](https://github.com/plugwise/python-plugwise-usb/commit/f6cc26d1104f072dfef298360470f26712fe1763)